src/pages/docs/creators/dapps/creating-golem-dapps.md

```diff
@@ -236,35 +229,35 @@ network: "default"
       - "192.168.0.4"
 ```
 
-#### `depends_on`
+#### this_sentence
```

Comment:
why 'this_sentence' ?

Reply:
That's a good question. I don't even remember doing it... - I will fix it.